tenant/store.go

```diff
@@ -41,7 +41,7 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 
 type Tenant struct {
 	ID         uint64 `sql:"AUTO_INCREMENT"`
```

Comment:
In Host Message, we have "Id". Here we have "ID". I know this is not part of this PR, but I'm just pointing it out. Would be nice to have this more standardized.